### PR TITLE
Fix dashboard mobile layout

### PR DESCRIPTION
**Pull request by aurlic**
### Changes

048cd4286255bb58b3617992b4d3160290b441e7 :
- tiny formatting adjustment one some lines
- started working on @media (max-width: 425px)
- made changes to the width to fix display bug

048cd4286255bb58b3617992b4d3160290b441e7 :
- Modification of some class name to better reflect role and avoid further overriding with other components

### Questions

- Should I fix and tweak the display any further ?
- If yes, should I do it for both different media size displays ?

### CODE REVIEW

File: frontend/src/pages/spotnet/dashboard/Dashboard.jsx [hunk 1]
```diff
@@ -120,7 +120,7 @@ const [isCollateralActive, setIsCollateralActive] = useState(true);
       <div className="dashboard-container">
         {loading && <Spinner loading={loading} />}
         <h1 className="dashboard-title">zkLend Position</h1>
-        <div className="main-container">
+        <div className="dashboard-content">
           <div className="top-cards">
             <div className="card">
```
Comment (whateverfw) [inline frontend/src/pages/spotnet/dashboard/Dashboard.jsx:125]: 2 cards should fit scren on 390px screen, you can see that currently their size is larger. Please check and implement according to figma

Comment (whateverfw) [reply frontend/src/pages/spotnet/dashboard/Dashboard.jsx:125]: Also currently cards has more padding the block below, but they should have the same width and padding (see Figma)

**Review by whateverfw — CHANGES_REQUESTED** [review]

**Review by whateverfw — APPROVED** [review]
LGTM

**Review by djeck1432 — APPROVED** [review]
Good job, looks good to me

**Pull request merged**